### PR TITLE
Add dockerflow middleware and circleci scripts

### PR DESCRIPTION
**Pull request by robotblake**

### CODE REVIEW

File: server/settings.py [hunk 1]
```diff
@@ -51,6 +51,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    'dockerflow.django',
```
Comment (wlach) [inline server/settings.py]: Could you make this optional, configured through an environment variable above? I don't think we want dockerflow for the heroku environment we're using.

File: server/settings.py [hunk 2]
```diff
@@ -78,6 +81,14 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
+if DOCKERFLOW_ENABLED:
+    INSTALLED_APPS.extend([
```
Comment (wlach) [inline server/settings.py]: I would just use `append` rather than `extend` here, though it's not that big a deal.

Comment (robotblake) [reply server/settings.py]: Oof, yeah, I'll make that change.

**Comment by robotblake** [comment]
Yeah, I'll wire it up before merging.

**Pull request merged**